### PR TITLE
AI Modal V2

### PR DESCRIPTION
**Pull request by chumpgrub**
Updates to the AI Intake Modal form.

Here's a Loom walkthrough of the UI changes: https://www.loom.com/share/fe3e225cca014283ac51f13c8e21a822

Updates Include:

Slider Examples
- Slides are now text-based, rather than image
- Background image changes based on example's industry

Industry Select Field
- Uses `react-select` package already included in repository
- Uses the `Creatable` component that will allow user to create value, if not found in our provided list
- Data drawn from the 8000+ industry list provided by Donaldo
  - I think we'd be better served by using an Elasticsearch-type solution for this amount of data

Location Input
- New card-based interface to better determine location type

About Textarea Input
- Adds recommended length indicator

Tone Input
- Uses updated tone list
- Uses `react-select` with custom option presentation

Keywords Input
- Minor usability adjustment

Photography Step
- Now defaults to "My Images" screen but the curated collections are still available for selection
- Minor style updates for better presentation on smaller screens

## 7/18/2023 Updates
- Updates wizard footer to better use primary/secondary actions
- Removes outdated AI intake user validation
- Content updates
- Style adjustments

### CODE REVIEW

File: src/plugins/prebuilt-library/ai-wizard/constants/index.js [hunk 1]
```diff
@@ -161,3 +136,49 @@ export const ENTITY_TO_NAME = {
...
+export const MISSION_STATEMENT_GOAL = 300;
+
+export const MISSION_STATEMENT_STATUS = {
+	'weak': {
+		color: '#DF3416',
+		message: __( 'For better, more accurate copy enter a bit more content.', 'kadence-blocks' ),
+	},
+	'medium': {
+		color: '#B35F00',
+		message: __( 'This is a great start. Try adding to your mission statement a bit more if you can.', 'kadence-blocks' ),
```
Comment (kadencewp) [inline src/plugins/prebuilt-library/ai-wizard/constants/index.js:176]: Can we update this to "This is a great start. Try adding a bit more if you can." That way we don't confuse by saying "mission statement"

File: src/plugins/prebuilt-library/ai-wizard/constants/index.js [hunk 2]
```diff
@@ -161,3 +136,49 @@ export const ENTITY_TO_NAME = {
...
+	{
+		icon: desktop,
+		text: __( 'Online Only', 'kadence-blocks' ),
+		value: LOCATION_ONLINE_ONLY,
+		help: '',
+		placeholder: '',
+	},
+];
+
+export const MISSION_STATEMENT_GOAL = 300;
```
Comment (kadencewp) [inline src/plugins/prebuilt-library/ai-wizard/constants/index.js:167]: 300 seams pretty short? Is there any reason behind 300? or can we make it 400 or 500 ?

**Review by kadencewp — CHANGES_REQUESTED** [review]
There are two small styling issues we need to address with WordPress 6.3

1. The main content is top-aligned.
https://share.getcloudapp.com/xQuEzWNn
2. The pagination buttons are not styled correctly. https://share.getcloudapp.com/9ZuyDEoD

File: src/plugins/prebuilt-library/ai-wizard/constants/index.js (hunk 2 above)
Comment (chumpgrub) [reply src/plugins/prebuilt-library/ai-wizard/constants/index.js:167]: Donaldo provided the following direction on the strength indicator: 

> Weak: <150 characters (~1 sentence long)
> Medium: 150-300 characters (~2 sentences long)
> Strong: >300 characters (~3+ sentences long)

I'm happy to ask about how we landed on 300 and ask about increasing the count in today's AI Scrum.

File: src/plugins/prebuilt-library/ai-wizard/constants/index.js (hunk 1 above)
Comment (chumpgrub) [reply src/plugins/prebuilt-library/ai-wizard/constants/index.js:176]: I've updated this content.

**Pull request merged**